### PR TITLE
[story/7] Display the details of an annoucement

### PR DESCRIPTION
**Pull request by Nicofuma**
issue: #20

### CODE REVIEW

**Comment by salahm** [comment]
please rebase

**Comment by Nicofuma** [comment]
all branch have been rebased

**Comment by salahm** [comment]
why are you still having `Connect integration` commit ?

**Comment by Nicofuma** [comment]
Because it is based on #39 which introduce the commit

File: src/SensioLabs/JobBoardBundle/Resources/views/Mail/updateNotification.html.twig [hunk 1]
```diff
@@ -1,26 +1,44 @@
+{% block subject %}A validated announcement has been updated{% endblock %}
+
+{% block body_txt %}
+Source: {{ path('job_show', {'country': new_job.country , 'contract': new_job.contractType, 'slug': new_job.slug }) }}
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/views/Mail/updateNotification.html.twig]: Why `path` ? This an email, the url needs to be absolute.

**Comment by ogizanagi** [comment]
Except my last comment, :+1:

File: src/SensioLabs/JobBoardBundle/Resources/views/Mail/updateNotification.html.twig (hunk 1 above)
Comment (Nicofuma) [inline src/SensioLabs/JobBoardBundle/Resources/views/Mail/updateNotification.html.twig]: updated

**Pull request merged**